README

```diff
@@ -1 +1,2 @@
 Hello World!
+this is octocat
```

Comment:
I think you should replace `octocat` with `Octocat` because it's a name. Also a period at the end of the sentence Is needed.